src/main/resources/application.conf

```diff
@@ -36,6 +36,13 @@ docker {
 // Optional call-caching configuration. Disabled by default.
 call-caching {
   enabled = false
+
+  // The Docker image specified in the 'runtime' section of a task can be used as-is
+  // or Cromwell can lookup this Docker image to get a complete hash.  For example,
+  // if a task specifies docker: "ubuntu:latest" and if lookup-docker-hash is true,
+  // Then Cromwell would query DockerHub to resolve "ubuntu:latest" to something like
+  // a2c950138e95bf603d919d0f74bec16a81d5cc1e3c3d574e8d5ed59795824f47
+  lookup-docker-hash = false
```

Comment:
Maybe just quickly say **why** you'd want to do this and what the effects of this would be